frontend/src/scenes/onboarding/CardContainer.tsx

```diff
@@ -1,33 +1,46 @@
 import React from 'react'
 import { Col, Card, Row } from 'antd'
 import { ArrowLeftOutlined } from '@ant-design/icons'
+import { Framework, PlatformType } from 'scenes/onboarding/types'
 
-export function CardContainer(props) {
+export function CardContainer({
+    index,
+    totalSteps,
+    onBack,
+    children,
+    nextButton,
+    onSubmit,
+}: {
+    index: number
+    totalSteps?: number
+    onBack?: () => void
+    children: React.ReactNode
+    nextButton?: boolean
+    onSubmit?: ({ type, framework }: { type?: PlatformType; framework?: Framework }) => void
```

Comment:
super nit: couldn't seem to find this func payload used anywhere so it could probably be changed to () => void also?

Reply:
that wasn't a nit but a valid point :). fixed!